msamp/megatron/layers.py

```diff
@@ -175,6 +196,9 @@ def backward(ctx, grad_output):
             wgrad_qtype,
             use_split_accumulator=True,
         )
+        if USE_W_DIFFERENTIABLE_GRADIENT_ESTIMATOR:
+            scaled_w = ctx.saved_tensors[0]
+            grad_weight.mul_(FP4_QUANT.apply_DGE_item(scaled_w, k=5.0, power_clamp_max=3.0))
```

Comment:
Let's just eliminate explicit argument assignments for `k` and `power_clamp_max` here, if their default values are already provided.